public/angular/templates/settings.html

```diff
@@ -25,13 +25,82 @@
       </tr>
     </tbody>
   </table>
+</div>
+<div ng-controller="MediaSettingsController">
+  <table class="table table-settings">
+    <thead>
+      <tr>
+        <td class="col-sm-3">
+          <label class="settings-label">Media</label>
+        </td>
+        <td>
+        </td>
+      </tr>
+    </thead>
+    <tbody>
+        <tr ng-repeat="mediaOption in mediaOptions">
+          <td class="col-sm-3">
+            <label class="settings-label">{{ mediaOption | capitalize }}</label>
+          </td>
+          <td>
+            <div data-toggle="buttons">
+              <label ng-click="toggle(mediaOption, false)"
+                     class="toggle-item ng-class:{'active':media[mediaOption].on===false}">
```

Comment:
Always use a two space indent please. Several instances in this file need to be fixed. Also please always use spaces around operators (`===`).